.pipelines/azure_pipeline_testframework.yaml

```diff
@@ -46,10 +46,12 @@ jobs:
 
             # Install TestKube CRs
             export WEBHOOK_URI=$(WEBHOOK_URI)
-            envsubst < ./testkube/testkube-teams-integration.yaml > ./testkube/testkube-teams-integration-$(CLUSTER_NAME).yaml
-            kubectl apply -f ./testkube/testkube-teams-integration-$(CLUSTER_NAME).yaml
+            export AZURE_CLIENT_ID="fea3fc3e-01b8-4af7-bf49-bd0815f09853"
```

Comment:
@suyadav1 - lets not expose AZURE_CLIENT_ID here and instead, lets pass through pipeline variable